common/locales/en/front.json

```diff
@@ -97,24 +97,24 @@
     "loginAndReg": "Login / Register",
     "loginFacebookAlt": "Login / Register with Facebook",
     "logout": "Log Out",
-    "marketing1Header": "Improve Your Habits By Playing A Game",
+    "marketing1Header": "Improve Your Habits by Playing a Game",
     "marketing1Lead1": "Habitica is a video game to help you improve real life habits. It \"gamifies\" your life by turning all your tasks (habits, dailies, and to-dos) into little monsters you have to conquer. The better you are at this, the more you progress in the game. If you slip up in life, your character starts backsliding in the game.",
-    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned",
+    "marketing1Lead2": "<strong>Get Sweet Gear</strong>. Improve your habits to build up your avatar. Show off the sweet gear you've earned!",
     "marketing1Lead2Title": "Get Sweet Gear",
-    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them, a system called \"stochastic rewarding\". Habitica accommodates all reinforcement styles: positive, negative, predictable, and random.",
+    "marketing1Lead3": "<strong>Find Random Prizes</strong>. For some, it's the gamble which motivates them: a system called \"stochastic rewards\". Habitica accommodates all reinforcement and punishment styles: positive, negative, predictable, and random.",
```

Comment:
I don't know, but I thought if we are using technical terminology (stochastic rewards), I would change it so that it uses reinforcement/punishment [correctly](https://en.wikipedia.org/wiki/Reinforcement#Terminology). If you think I should change it, I will.